packages/contracts-bedrock/contracts/L1/OptimismPortal.sol

```diff
@@ -189,8 +189,19 @@ contract OptimismPortal is Initializable, ResourceMetering, Semver {
         // and to prevent replay attacks.
         bytes32 withdrawalHash = Hashing.hashWithdrawal(_tx);
 
+        // Load the ProvenWithdrawal into memory
+        ProvenWithdrawal memory provenWithdrawal = provenWithdrawals[withdrawalHash];
+
+        // Only allow re-proving a withdrawal transaction if the output root has changed.
+        require(
+            provenWithdrawal.timestamp == 0 ||
+                (_l2BlockNumber == provenWithdrawal.l2BlockNumber &&
+                    outputRoot != provenWithdrawal.outputRoot),
+            "OptimismPortal: withdrawalHash has already been proven"
```

Comment:
Nit.
I just checked, and most often our error messages use regular english rather than variable names.

```suggestion
            "OptimismPortal: withdrawal hash has already been proven"
```